common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal_types.h

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2016-2019 Intel Corporation
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#ifndef _BWL_MON_WLAN_HAL_DWPAL_TYPES_H_
+#define _BWL_MON_WLAN_HAL_DWPAL_TYPES_H_
+
+namespace bwl {
+namespace dwpal {
+
+#define NL_MAX_REPLY_BUFFSIZE 8192
+#define NL_ATTR_HDR 4
+
+struct sScanCfgParams {
+    int passive;
```

Comment:
Why do you need both `passive` and `active`? can we do a scan which is both passive and active?

Reply:
Also here and in other structs defined in this file I think we should change to use `bool` and `size_t` instead of `int`.

Reply:
Finally, please add a doxygen comment for each struct which defines what this struct is used for. And for each member of each struct, add a comment which explains its meaning.

Reply:
renamed passive/active_dwell_time.
added comments,